gpxpy/parser.py

```diff
@@ -102,7 +102,7 @@ def parse(self, version=None):
         schema_loc = mod_re.search(r'\sxsi:schemaLocation="[^"]+"', self.xml)
         if schema_loc:
             _, _, value = schema_loc.group(0).partition('=')
-            self.gpx.schema_locations = value.strip('"')
+            self.gpx.schema_locations = value.strip('"').split(' ')
```

Comment:
In case there are multiple spaces, maybe use a simple no-argument `split()`?

Reply:
Done.